src/Transition.js

```diff
@@ -52,10 +52,12 @@ export default class Transition extends React.PureComponent {
     children: PropTypes.oneOfType([
       PropTypes.arrayOf(PropTypes.func),
       PropTypes.func,
+      PropTypes.bool,
```

Comment:
@lasseborly  `{visible && fn}` isn't a boolean, though. It either results in a function of undefined.